ProfileManager.js

```diff
@@ -382,15 +382,15 @@ export default class ProfileManager {
       return;
     }
 
-    // cache account capability agent
-    const {secret} = (authentication || {});
-    this.capabilityAgent = await (secret ?
-      _getCapabilityAgentFromSecret({secret, accountId}) :
-      CapabilityAgent.fromCache({handle: accountId}));
-    if(this.capabilityAgent === null) {
-      // could not load from cache and no `secret`, so cannot load edv
-      return;
-    }
+    const crypto = (self.crypto || self.msCrypto);
+    const TextDecoder = self.TextDecoder;
+    const buffer = new Uint8Array(32);
+    crypto.getRandomValues(buffer);
+    const secret = new TextDecoder('utf-8').decode(buffer);
```

Comment:
There's no guarantee that random bytes are a properly UTF-8 encoded string (they are random). But we don't need this anyway because we shouldn't be using any salt from the server as we do in that helper function below. We should remove that helper function entirely and just call `CapabilityAgent.fromSecret` and pass the `secret` in as a `Uint8Array`.

Reply:
JFYI, to make the TextDecoder throw an error you have to do: `new TextDecoder('utf8', {fatal: true})` ... otherwise it doesn't throw ... and who knows what it actually does, but if you try to encode again you won't get the same thing. I don't know what that does to the randomness so it shouldn't be used (again, we don't have to worry about it here, this is just informational for the future).